rsc/src/main/scala/rsc/semanticdb/Writer.scala

```diff
@@ -66,12 +66,22 @@ final class Writer private (
         reporter.append(DefnMethodNotype(defn, settings.notypeWarn))
 
       case (defn: DefnClass, _: s.ClassSignature) =>
-        defn.parents match {
+        val maybeInitId = defn.parents match {
           // Parent Init does not have type parameters specified, so make sure none are needed.
           // Had type parameters been specified, tpt would be of type TptParameterize
           // We will rely on scalac to deal with any further type parameter errors
```

Comment:
It might be a little bit clearer to name move this comment just outside of the `match` block and rename `maybeInitId` to `maybeUnparameterizedInitId` so the `.foreach` block below is more self-explanatory.

Reply:
I typically like to err on the side of shorter names but that might be a mistake